src/components/interactive-builder/interactive-builder.component.tsx

```diff
@@ -204,15 +209,134 @@ const InteractiveBuilder: React.FC<InteractiveBuilderProps> = ({
     [onSchemaChange, schema, t]
   );
 
+  const handleFormValidation = () => {
+    setResponses([]);
+
+    if (schema) {
+      const parsedForm =
+        typeof schema == "string" ? JSON.parse(schema) : schema;
+
+      for (let i = 0; i < parsedForm.pages.length; i++) {
+        for (let j = 0; j < parsedForm.pages[i].sections.length; j++) {
+          for (
+            let k = 0;
+            k < parsedForm.pages[i].sections[j].questions.length;
+            k++
+          ) {
+            const questionObject = parsedForm.pages[i].sections[j].questions[k];
+
+            if (questionObject.type === "obsGroup") {
+              questionObject.questions.forEach((question) => {
+                handleQuestionValidation(question);
+              });
+            }
+            handleQuestionValidation(questionObject);
+          }
+        }
+      }
+    } else {
+      console.log("Empty form!");
```

Comment:
@arodidev since you are now showing the errors / warnings on the screen why don't you also show this as an wringing on the screen. Am saying this because one can just think the button is not working if they don't look at the console logs.